docs/schemas/fileDownload.json

```diff
@@ -18,7 +18,12 @@
                 "application/rtf",
                 "text/plain",
                 "image/png",
+                "image/jpeg",
+                "image/gif",
                 "text/html"
```

Comment:
Looks like I missed a comma :(

@abraae what's the best way to fix this? Commit straight to master? Open a new PR?

Reply:
Its OK, I fixed it in the PR I just merged, which renames your picklist to cascade, and re-introduces a simple picklist